app/uk/gov/hmrc/gform/gform/NewFormController.scala

```diff
@@ -492,9 +492,13 @@ class NewFormController(
     l: LangADT
   ): Future[Result] =
     for {
-      itmpRetrievals <- formTemplate.dataRetrieve.fold(Future.successful(Option.empty[ItmpRetrievals]))(_ =>
-                          auth.getItmpRetrievals(request).map(Some(_))
-                        )
+      itmpRetrievals <- if (formTemplate.isSpecimen) {
+                          Future.successful(Option.empty[ItmpRetrievals])
```

Comment:
does it work if some fields has includeIf that depends on itmpRetrievals or is it just ignores includeIf when in specimen template mode?

Reply:
When posting a template, it removes all includeIfs if the template is a specimen template. It's working as expected.

Reply:
I think we don't need to call  the external APIs for the specimen template.

Reply:
Shouldn't we remove `dataRetrieve` from specimen instead?